versioned_docs/version-23.04/administration/backup-poller.md

```diff
@@ -0,0 +1,52 @@
+---
+id: backup-poller
+title: Back up and restore your pollers
+---
+
+## Case n°1: Making backups to be able to install a new poller from scratch
+
+### What you need to back up on your active poller
+
+You need to back up the following elements to be able to rebuild your poller from scratch in case of a problem.
```

Comment:
```suggestion
> In this list, we assume you have enabled the [automatic installation of the Centreon plugins](../monitoring/pluginpacks.md#automatic-installation).

You need to back up the following elements to be able to rebuild your poller from scratch in case of a problem.
```